packages/devextreme/testing/tests/DevExpress.ui.widgets/contextMenu.tests.js

```diff
@@ -393,6 +395,63 @@ QUnit.module('Rendering ScrollView', moduleConfig, () => {
         assert.strictEqual($submenu.outerHeight(), $(window).height(), 'menu uses the full height of the window');
         assert.strictEqual($submenu.offset().top, 0, 'menu does not cross the window border');
     });
+
+    QUnit.test('Selected item should be always visible during keyboard navigation (root menu)', function(assert) {
+        if(!isDeviceDesktop(assert)) {
+            return;
+        }
+
+        const instance = new ContextMenu(this.$element, {
+            items: (new Array(99)).fill(null).map((_, idx) => ({ text: `item ${idx}` })),
+            focusStateEnabled: true,
+            visible: true,
+        });
+        const $scrollableContainer = $(instance.itemsContainer()).find(`.${DX_SCROLLABLE_CONTAINER_CLASS}`);
+        const $scrollableContent = $(instance.itemsContainer()).find(`.${DX_SCROLLABLE_CONTENT_CLASS}`);
+
+        assert.strictEqual($scrollableContent.position().top, 0, 'initial position');
+
+        keyboardMock(instance.itemsContainer())
+            .press('up')
+            .press('up');
+
+        assert.roughEqual($scrollableContent.position().top,
+            $scrollableContainer.height() - $scrollableContent.height(), 2, 'scrolled to bottop');
```

Comment:
```suggestion
            $scrollableContainer.height() - $scrollableContent.height(), 2, 'scrolled to bottom');
```
Please fix in all places